frontend/src/components/Navbar.jsx

```diff
@@ -109,7 +109,13 @@ function Navbar() {
           onClick={closeMobileMenu}
         >  Create Post </Link>
       </li>
-
+      <li class='item-right'>
```

Comment:
Comments component is used for Post.jsx module. It should not appear in the Navbar section (Navbar.jsx). Please remove it.